creator-node/package.json

```diff
@@ -5,6 +5,9 @@
   "scripts": {
     "start": "nodemon src/index.js | ./node_modules/.bin/bunyan || exit 0",
     "test": "./scripts/run-tests.sh",
+    "test:local": "./scripts/run-tests.sh standalone_creator",
```

Comment:
i think it would be most intuitive to have the base `npm test`. run. whatever test:local:teardown is doing. locally i'd expect it to work and cleanup without other options as the. default. and test can become `test-ci`. and `test:local` could be `test:local:noteardown`